src/app/screens/LNURLPay/index.tsx

```diff
@@ -518,7 +524,7 @@ function LNURLPay() {
                         isFocused={false}
                         label={tCommon("actions.confirm")}
                         loading={loadingConfirm}
-                        disabled={loadingConfirm || !valueSat}
+                        disabled={loadingConfirm || !valueSat || amountExceeded}
```

Comment:
Should the `rangeExceeded` be included in here as well?

Reply:
Updated.